### ISSUE TITLE
Train loss

### ISSUE THREAD

**Issue by michaelscheinfeild** [pavelsuma/ames#12]
need fix  where no teacher

        if beta > 0:
         loss = loss + beta * dist_loss(student_logits, teacher_logits).mean()

in training.py  line 69~
